generator/src/genclient/genclient.go

```diff
@@ -84,8 +84,7 @@ func (r *GenerateRequest) outDir() string {
 
 // Output of generation.
 type Output struct {
-	// TODO(codyoss): needs enough info to pass information back to protoc
-	// on what was generated.
+	// TODO(codyoss): https://github.com/googleapis/google-cloud-rust/issues/32
```

Comment:
I usually write TODO(#32) and maybe a short summary. Your call.

Reply:
oops, had auto merge on this. Ack for next time